TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/datahandlers/NettyPacketToHttpConsumerTest.java

```diff
@@ -315,6 +316,90 @@ public void testMetricCountsFor_testThatConnectionsAreKeptAliveAndShared(boolean
         Assertions.assertEquals(2, connectionsClosedCount);
     }
 
+    @ParameterizedTest
+    @CsvSource({"false", "true"})
+    @Tag("longTest")
+    public void testResponseTakesLongerThanTimeout(boolean useTls) throws Exception {
+        var testDurationBuffer = Duration.ofSeconds(10);
```

Comment:
nit: rename this to something like "max time for test to wait for read timeout or response"?
You could also put it closer to the get() call to bind the application and value to make the intent clearer.